src/queries.js

```diff
@@ -1,11 +1,25 @@
 // TODO: Sanitize string input of buckets
 
+// Helper function returning a query that defines a variable events_active
+// that can be used to filter away non-active time.
+function events_active(afkbucket, audibleAsActive) {
+  return [
+    'events_active = [];',
+    `events_afk = flood(query_bucket("${afkbucket}"));`,
+    'events_active = period_union(events_active, filter_keyvals(events_afk, "status", ["not-afk"]));',
+  ].concat(audibleAsActive ? [
+    'events_audible = filter_keyvals(events_browser, "audible", [true]);',
```

Comment:
Does `[true]` work?

Reply:
Just experimented with a similar query in aw-research (https://github.com/ActivityWatch/aw-research/pull/12) and no, it doesn't.

Reply:
1 or TRUE might work though, test that.

Reply:
Added boolean support in https://github.com/ActivityWatch/aw-core/pull/70/commits/3c4187e50bffb60a16593d671dbe4b301453e6e4